src/renderer/components/SettingsPage.jsx

```diff
@@ -203,13 +203,13 @@ export default class SettingsPage extends React.PureComponent {
             key: 'notifications',
             data: {
                 ...this.state.notifications,
-                flashWindow: this.flashWindowRef.current.props.checked ? 2 : 0,
```

Comment:
Did we test this on all OSes? This was breaking for me on my Windows 10 machine for some reason.